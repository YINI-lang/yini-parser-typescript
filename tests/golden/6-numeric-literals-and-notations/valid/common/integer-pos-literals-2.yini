// Common positive edge-case decimal integers.
^ PosInts
int1 = 0
int2 = 1                  // smallest positive integer
int3 = 2147483647         // max 32-bit signed integer
int4 = 9223372036854775807 // max 64-bit signed integer
int5 = 999999999          // large positive but still within 32-bit range

^ PosInts2
int1 = +0
int2 = +1                  // smallest positive integer
int3 = +2147483647         // max 32-bit signed integer
int4 = +9223372036854775807 // max 64-bit signed integer
int5 = +999999999          // large positive but still within 32-bit range
