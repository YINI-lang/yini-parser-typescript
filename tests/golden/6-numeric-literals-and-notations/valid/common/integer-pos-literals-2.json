{
    "PosInts": {
        "int1": 0,
        "int2": 1,
        "int3": 2147483647,
        "int4": 9223372036854775807,
        "int5": 999999999
    },
    "PosInts2": {
        "int1": 0,
        "int2": 1,
        "int3": 2147483647,
        "int4": 9223372036854775807,
        "int5": 999999999
    }
}
